prime-dotnet-webapi/Services/AccessTermService.cs

```diff
@@ -64,12 +64,14 @@ public async Task CreateEnrolleeAccessTermAsync(Enrollee enrollee)
 
         public async Task SetAcceptedDateForAccessTermAsync(Enrollee enrollee)
         {
-            var termsOfAccess = await _context.AccessTerms
+            var accessTerm = await _context.AccessTerms
                 .Where(toa => toa.EnrolleeId == enrollee.Id)
                 .OrderByDescending(toa => toa.AcceptedDate)
                 .FirstOrDefaultAsync();
 
-            termsOfAccess.AcceptedDate = DateTime.Now;
+            accessTerm.AcceptedDate = DateTime.Now;
+            // Add an Expiry Date of one year in the future.
+            accessTerm.ExpiryDate = DateTime.Now.AddYears(1);
```

Comment:
What should happen if this method gets called when the most recent access term is already accepted? Programatically this makes total sense but accepting the TOA is akin to signing a legal document so changing this date seems off to me.

Reply:
So the case is that a user has accepted a the most recent TOA, and somehow a user receives a TOA that was created before that one and accepts it?

Are you alluding to we should check that if a accessTerm already has a accepted + expiry date we should not override them?

Reply:
This method could (theoretically) be called from anywhere in our code. It _currently_ is only called when the enrollee goes to the ACCEPTED_TOA status, but even then we aren't checking that they actually have a non-accepted TOA to "sign", we are 100% trusting in the status change process. Which feels a little iffy to me when it represents signing a legal document.

I feel like if the newest TOA has an accepted date (is already signed) this should be another loud error.

Reply:
Alright i see where you are coming from. Should i move those thoughts into a technical backlog tickets as that seems out of the scope of this ticket. Adding an expiry date and having it displayed on the front end. This would be more changing the logic on how the code handles the TOA.

Reply:
https://bcgovmoh.atlassian.net/browse/PRIME-320?atlOrigin=eyJpIjoiNDdkY2JmMTZmNjMyNDRjZTlhZWRhN2FiMTU2MzhkMWEiLCJwIjoiaiJ9 for reference